modules/oadp-creating-restore-hooks.adoc

```diff
@@ -53,23 +53,25 @@ spec:
               command:
               - /bin/ash
               - -c
+            timeout: <4>
         - exec:
-            container: <container> <4>
+            container: <container> <5>
             command:
-            - /bin/bash <5>
+            - /bin/bash <6>
             - -c
             - "psql < /backup/backup.sql"
-            waitTimeout: 5m <6>
-            execTimeout: 1m <7>
-            onError: Continue <8>
+            waitTimeout: 5m <7>
+            execTimeout: 1m <8>
+            onError: Continue <9>
 ----
 <1> Optional: Array of namespaces to which the hook applies. If this value is not specified, the hook applies to all namespaces.
-<2> Currently, pods are the only supported resource.
+<2> Currently, pods are the only supported resource that hooks can apply to.
 <3> Optional: This hook only applies to objects matching the label selector.
-<4> Optional: If the container is not specified, the command runs in the first container in the pod.
-<5> Array of commands that the hook runs.
-<6> Optional: If the `waitTimeout` is not specified, the restore waits indefinitely. You can specify how long to wait for a container to start and for preceding hooks in the container to complete. The wait timeout starts when the container is restored and might require time for the container to pull the image and mount the volumes.
-<7> Optional: How long to wait for the commands to run. The default is `30s`.
-<8> Allowed values for error handling are `Fail` and `Continue`:
+<4> Timeout specifies the maximum amount of time Velero waits for `initContainers` to complete.
+<5> Optional: If the container is not specified, the command runs in the first container in the pod.
+<6> Array of commands that the hook runs.
```

Comment:
@mperetzred 
entrypoint array of commands?

Reply:
no, just entrypoint array. The entrypoint is the "task" that the container is running, and once it completes, the container is terminated (or not; it could be a command that doesn't return, and then the container will run forever). 

More details about entrypoints:
https://docs.docker.com/engine/reference/builder/#entrypoint

I'm not sure we need to give an explicit explanation on our doc what it's an entrypoint on a container.

The upstream velero also refers it as an entrypoint:
https://velero.io/docs/v1.9/restore-hooks/#:~:text=This%20is%20the%20ENTRYPOINT%20for%20the%20init%20container%20being%20added

so maybe we could use the same description as in the velero docs to avoid complications/misunderstandings

Reply:
Done